service/src/main/java/bio/terra/tanagra/app/WebMvcConfig.java

```diff
@@ -17,6 +17,6 @@ public WebMvcConfig(AuthInterceptor authInterceptor) {
 
   @Override
   public void addInterceptors(InterceptorRegistry registry) {
-    registry.addInterceptor(authInterceptor);
+    registry.addInterceptor(authInterceptor).addPathPatterns("/v2/**", "/status");
```

Comment:
I think it would be safer to require auth everywhere by default (`/**`?) and only exclude specific URLs if we need to.

Reply:
Good idea. Changed to intercept all paths and added an exception for the static swagger resources that let us hit endpoints manually in a webpage.